### ISSUE TITLE
Read configurations from different files

### ISSUE THREAD

**Issue by VincenzoFerme** [benchflow/benchflow#142]
_From @VincenzoFerme on February 11, 2016 8:5_

We are moving some configuration variables to be handled by Consul, and serialised to file by envconsul. This means we need to read also another `*.yml` file (for now at boot time of the application, since we do not expect it to be updated), and store the information in the Dropwizard way.

Location of the file in the container:
- /app/config.yml

A reference to start with for reading information also from this file:
- http://stackoverflow.com/questions/28727224/can-i-have-multiple-configuration-files-in-dropwizard

A reference for future work, where we should monitor the changes to the file and update the configuration stored in memory:
- https://github.com/ReactiveX/RxJava

Once it is done, remove the MINIO variables also from the deployment descriptor of the service.

_Copied from original issue: benchflow/drivers-maker#20_
